### DISCUSSION TITLE
伤心

### DISCUSSION THREAD

**Discussion by Jason555647** [share121/inter-knot#5331] [灌水]
![122649626_p0](https://github.com/user-attachments/assets/696f4d49-a1fc-4255-8aed-92ee19dbfedc)
似乎喜欢可琳的很少呢( ´•︵•` )

**Comment by Loxia1121** [comment]
与其说喜欢可琳的人不多，倒不如说只喜欢可琳的人很少（）我平时的Xp是帅哥大姊姊什么的 但我也觉得可琳这种可爱妹妹挺招人喜欢的（在无聊的时候逛街 遇到可琳会帮帮她然后被小女仆狠狠可爱到（！！！）

**Comment by RinzeExp** [comment]
可琳厨来了！！完全无法拒绝这种软软的可爱生物！

**Comment by hapildk** [comment]
大概是因为喜欢可琳的都和可琳一样没有自信展示自己的厨力

**Comment by LeoneMafia** [comment]
可琳厨前来报到！
